addons/kommander/1.x/kommander.yaml

```diff
@@ -74,3 +74,6 @@ spec:
           repository: mesosphere/kubeaddons-catalog
           tag: "v0.8.2"
           pullPolicy: IfNotPresent
+      
+      kommander-ui:
+        kubernetesVersionsSelection: '["1.16.4", "1.16.3"]'
```

Comment:
Please bump the revision for the addon, and then :+1:

Reply:
How do I update the addon revision?

Reply:
> catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-14"

becomes

> catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-15"